packages/help-center/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
 	preset: '../../test/packages/jest-preset.js',
+	setupFiles: [ '<rootDir>/jestSetup.ts' ],
 	testEnvironment: 'jsdom',
+	moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx', 'json', 'node' ],
+	transformIgnorePatterns: [ 'node_modules/(?!react-native|react-navigation)/' ],
```

Comment:
```suggestion
	transformIgnorePatterns: [ 'node_modules/(?!gridicons)(?!.*\\.svg)' ],
```
I think we shouldn't be concerned with react-native and react-navigation.
I see there is an issue with the svgs coming from gridicons. This config will fix it.

Reply:
I read up more on the transformIgnorePatterns configuration and your changes make perfect sense 👍